### PR TITLE
Move lmnet/lmnet/data_augmentor.py to blueoil/data_augmentor.py

### PR DESCRIPTION
**Pull request by iizukak**
This PR is a part of #796
Move lmnet/lmnet/data_augmentor.py to blueoil/data_augmentor.py

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: output_template/python/lmnet/data_augmentor.py [hunk 1]
```diff
@@ -1 +0,0 @@
-../../../lmnet/lmnet/data_augmentor.py
```
Comment (suttang) [inline output_template/python/lmnet/data_augmentor.py]: Need to create `output_template/python/blueoil/data_augmentor.py` ?

Comment (iizukak) [reply output_template/python/lmnet/data_augmentor.py]: @suttang 
In `output_template`, We don't use `data_augmentor`.

Comment (iizukak) [reply output_template/python/lmnet/data_augmentor.py]: I don't know why anyone create this symbolic link.

Comment (suttang) [reply output_template/python/lmnet/data_augmentor.py]: OH! TASHIKANI!

**Pull request merged**